manager/controllers/baseline_detail.go

```diff
@@ -15,7 +15,8 @@ import (
 )
 
 type BaselineDetailResponse struct {
-	Data BaselineDetailItem `json:"data"`
+	Data        BaselineDetailItem `json:"data"`
+	Description string             `json:"description,omitempty"`
```

Comment:
Should not be `Description` part of `Data`, actually part of `BaselineDetailAttributes` type?

Reply:
Probably yes

Reply:
I changed that. Looks much better now :slightly_smiling_face: Thanks for catching this.